community.json

```diff
@@ -1692,6 +1692,7 @@
     },
     "wikijs": {
         "branch": "master",
+        "level": 5,
```

Comment:
Like for distbin, the CI hasn't updated the url from community.json, still using the old address https://github.com/yalh76/wikijs_ynh

Update of the community.json made 8 days ago ^^